mbed/mbed_cdc.py

```diff
@@ -0,0 +1,111 @@
+
+def mbed_cdc(port, reset=False, sterm=False, baudrate=9600, timeout= 10, print_term_header=True):
```

Comment:
Could we drop `mbed` from the name of this function? it adds no meaning here. I think this is more a `run_serial_term` or something similar.

Reply:
But it's not a standard serial terminal and rather for Mbed needs. So I'd rather not use a generic name. Also the whole file would rather go in https://github.com/ARMmbed/mbed-sterm, and this function name will be the contract.

Reply:
Understood. This is resolved.